tools/vt/vt_normalize.xml

```diff
@@ -57,7 +57,7 @@
             <when value="cached">
                 <param name="reference_genome" type="select" label="Using reference genome">
                     <options from_data_table="fasta_indexes">
-                        <filter type="data_meta" ref="infile" key="dbkey" column="1" />
+                        <filter type="sort_by" column="2" />
```

Comment:
Strange, then it might be something on our end. 
Our fasta_indexes table looks like this:
```
GRCh37	GRCh37	H.sapiens (hg19/GRCh37)	/exports/genomes/species/H.sapiens/GRCh37/reference.fa
WBcel235	WBcel235	C.elegans (WBcel235)	/exports/genomes/species/C.elegans/WBcel235/reference.fa
Rnor_6.0	Rnor_6.0	R.norvegicus (Rnor_6.0)	/exports/genomes/species/R.norvegicus/Rnor_6.0/reference.fa
GRCm38	GRCm38	M.musculus (GRCm38)	/exports/genomes/species/M.musculus/GRCm38/reference.fa
TAIR10	TAIR10	A. Thaliana (TAIR10)	/exports/genomes/species/A.thaliana/TAIR10/reference.fa
GRCh38	GRCh38	H.sapiens (hg38/GRCh38)	/exports/genomes/species/H.sapiens/GRCh38/reference.fa
CanFam3.1	CanFam3.1	C.l.familiaris (CanFam3.1)	/exports/genomes/species/C.l.familiaris/CanFam3.1/reference.fa
P_pygmaeus_2_0_2	P_pygmaeus_2_0_2	P.abelii (P_pygmaeus_2_0_2)	/exports/genomes/species/P.abelii/P_pygmaeus_2_0_2/reference.fa
GRCz10	GRCz10	D.rerio (GRCz10)	/exports/genomes/species/D.rerio/GRCz10/reference.fa
BDGP6	BDGP6	D.melanogaster (BDGP6)	/exports/genomes/species/D.melanogaster/BDGP6/reference.fa
```